src/main/resources/anniv5Resources/localization/eng/calamitypack/Cardstrings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "${ModID}:Doomtouch": {
+    "NAME": "Doomtouch",
+    "DESCRIPTION": "Deal !D! damage. NL Apply !M! ${ModID}:Jinx."
+  },
+  "${ModID}:BindingChains": {
+    "NAME": "Binding Chains",
+    "DESCRIPTION": "Gain !B! Block. NL Enemy loses !M! Strength this turn. NL Exhaust."
+  },
+  "${ModID}:FreezingVenom": {
+    "NAME": "Freezing Venom",
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL Apply !M! ${ModID}:Frostbite and Poison to ALL enemies."
```

Comment:
Bit of a mismatch between the description and the actual effect.
Would recommend doing the normal AoE damage here and then using the AllEnemyApplyPowerAction to apply the powers. This would also speed up the distribution.

Otherwise, I'd ask you to reword this to say each enemy, not ALL enemies.